Makefile

```diff
@@ -151,7 +151,6 @@ tidy:
 
 .PHONY: check
 check: fmtcheck vet
-	@$(MAKE) --no-print-directory tidy
```

Comment:
intentional?

Reply:
`make tidy` fails for 1.15 but not 1.16, and make tidy is already called during the deps job so I thought it wouldn't be necessary to run it again for the older versions.